h/accounts/test/views_test.py

```diff
@@ -391,10 +393,15 @@ def test_forgot_password_sends_mail(reset_mail, authn_policy, mailer):
     controller = ForgotPasswordController(request)
     controller.form = form_validating_to({"user": user})
     message = reset_mail.return_value
+    reset_mail.return_value = {
+        'recipients': ['giraffe@thezoo.org'], 'subject': 'subject',
```

Comment:
This looks suspiciously like YAPF-formatted code to me ;)

I find this really hard to read. Any of

1) all on one line
2) 
```python
foo = {
    'bar': 'baz',
    'wibble': 'nonsense',
}
```
3)
```python
foo = {'bar': 'baz',
       'wibble': 'nonsense'}
```
is absolutely fine, but not some mixture of the three!